datacore/__init__.py,sha256=L1s4ZERgegBengHRGjP3jXrxroz7qhu44dAr6L0gbg4,285
datacore/datacore_restlike.py,sha256=HNJHlGKnbplbOsGO25DxF6s1X9mzfVQazy24LJMQMEI,49484
datacore/exceptions.py,sha256=v-RlUVvbFjWwTqcFuTjVn4VEhAOArTA4u_HJIY3rj1Y,2538
datacore/types.py,sha256=zfvfJGeTEH7rR3mnweMa4bE_5tHgc0L2zxurchcaosA,1263
datacore_python_client-1.0.2.dist-info/DESCRIPTION.rst,sha256=OCTuuN6LcWulhHS3d5rfjdsQtW22n7HENFRh6jC6ego,10
datacore_python_client-1.0.2.dist-info/METADATA,sha256=nC9S8mRkcD5pN3LmqGcfjXLc_g0hQEAsSOIviif7pHQ,441
datacore_python_client-1.0.2.dist-info/RECORD,,
datacore_python_client-1.0.2.dist-info/WHEEL,sha256=rNo05PbNqwnXiIHFsYm0m22u4Zm6YJtugFG2THx4w3g,92
datacore_python_client-1.0.2.dist-info/metadata.json,sha256=GwHvOGl33-wzX0B1_qP8G4AfbcUaGj2Nl9MVfk01_po,643
datacore_python_client-1.0.2.dist-info/top_level.txt,sha256=Rm_HpdZaQHkZIHrz7jvVe5kCOvOYmffPHK1pJ0G9hpI,9
